src/modules/esl-media/core/esl-media.ts

```diff
@@ -335,6 +346,28 @@ export class ESLMedia extends ESLBaseElement {
     this.deferredReinitialize();
   }
 
+  @listen({
+    event: 'esl:show',
+    target: window
+  })
+  protected _onContainerShow(e: Event): void {
+    const {target} = e;
+    if (!isSafeContains(target as Node, this)) return;
+    if (this.playInViewport && isVisible(this)) return;
```

Comment:
Probably here we should have the following condition
```suggestion
    if (this.playInViewport && !isVisible(this)) return;
```